Ys cb eedr update version (#34757)
demisto/content: Packs/CarbonBlackEnterpriseEDR/ReleaseNotes/1_1_34.md

#### Integrations

##### Carbon Black Enterprise EDR

- Updated the integration to match the new API version (v7) of Carbon Black.

>   Note: Breaking Change- The 'cb-eedr-list-alerts' command has no support to 'alert_category', 'workflow' and 'group_results' fields.

Reviewer: ```suggestion
  Note: Breaking Change - The ***cb-eedr-list-alerts** command not longer supports the *alert_category*, *workflow*, and *group_results* fields.
```

#### Playbooks

##### Carbon Black EDR Search Process

> - Deprecated. Use 'cb-eedr-process-search' command instead.

Reviewer: ```suggestion
Deprecated. Use ***cb-eedr-process-search*** command instead.
```
